### PR TITLE
feat: create a bot card feature

### LINKED ISSUE #72: 🎨 Add a "Create bot" option if user has no bots yet
## User stories

A user story is implemented as well as it is communicated.
If the context and the goals are made clear, it will be easier for everyone to implement it, test it, refer to it…

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Summary

On the user home page (bot dashboard), the page currently looks empty when no bots are created yet. As depicted below:

image

To increase my user experience, as a user I'd like to be guided by a larger and more inviting card in case I have no bots, as depicted below:

image

### PR DESCRIPTION
**Pull request by kelvinmburu**
This feature has been implemented by adding a card with a modal on the home screen. The card allows users to create a new bot if they don't have one.

Issue solved: https://github.com/italanta/elewa/issues/72

Fixes # (issue) - Fixes #72 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (Before and After)
## Before
elewa_b

## After
Elewa-Conversational-Learning-Manager

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code

### CODE REVIEW

File: libs/features/convs-mgr/home/src/lib/components/story-list/story-list.component.ts [hunk 1]
```diff
@@ -10,18 +10,23 @@ import { faSearch } from '@fortawesome/free-solid-svg-icons';
...
   filterInput$$: BehaviorSubject<string> = new BehaviorSubject<string>('');
 
   searchIcon = faSearch;
 
-  constructor(private _stories$$: StoriesStore) {
+  constructor(
+    private _stories$$: StoriesStore,
+
+  ) {
     this.stories$ = this._stories$$.get();
```
[the remark this answers is not in the document]
Comment (kelvinmburu) [reply libs/features/convs-mgr/home/src/lib/components/story-list/story-list.component.ts:23]: @Lucywambui Does the requested change have to do with indentation and removing the extra ','?

**Comment by kelvinmburu** [comment]
I've implemented the changes requested

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Resolve conflicts on this branch by merging main into your branch.

**Comment by kelvinmburu** [comment]
@IanOdhiambo9 Conflict resolved

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Great!.
Your sidemenu appears to be different from the one on main, can you check?

**Comment by kelvinmburu** [comment]
@IanOdhiambo9 By the time I made the PR, I hadn't pulled the new changes done on the side menu to my branch

File: libs/features/convs-mgr/home/src/lib/pages/home/home.page.html [hunk 2]
```diff
@@ -10,7 +10,15 @@ <h2>{{ org.name }} bots</h2>
...
-  <convl-italanta-apps-story-list fxFlexFill>
+  <convl-italanta-apps-story-list>
   </convl-italanta-apps-story-list>
+
+  <div class="mat-card-cvr">
+    <mat-card class="card" (click)="openDialog()">
+      <mat-card-title><i class="fa fa-plus fa-4x" aria-hidden="true"></i>
+      </mat-card-title>
+      {{ 'BOTS.HOME.ACTIONS.LBL-NEW-BOT' | transloco }}
+    </mat-card>
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/home/src/lib/pages/home/home.page.html:21]: This card should appear if the user has no stories yet!!

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Resolve conflicts

**Pull request closed without merging**